Formula/librasterlite2.rb

```diff
@@ -0,0 +1,101 @@
+class Librasterlite2 < Formula
+  desc "Library to store and retrieve huge raster coverages"
+  homepage "https://www.gaia-gis.it/fossil/librasterlite2/index"
+  url "https://www.gaia-gis.it/gaia-sins/librasterlite2-sources/librasterlite2-1.1.0-beta1.tar.gz"
+  sha256 "f7284cdfc07ad343a314e4878df0300874b0145d9d331b063b096b482e7e44f4"
+  license any_of: ["MPL-1.1", "GPL-2.0-or-later", "LGPL-2.1-or-later"]
+
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "freexl"
+  depends_on "geos"
+  depends_on "giflib"
+  depends_on "jpeg"
+  depends_on "libgeotiff"
+  depends_on "libpng"
+  depends_on "librttopo"
+  depends_on "libspatialite"
+  depends_on "libtiff"
+  depends_on "lz4"
+  depends_on "minizip"
+  depends_on "openjpeg"
+  depends_on "pixman"
+  depends_on "proj@7"
+  depends_on "sqlite"
+  depends_on "webp"
+  depends_on "xz"
+  depends_on "zstd"
+
+  uses_from_macos "curl"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  # Fix -flat_namespace being used on Big Sur and later.
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
+    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
+  end
+
+  def install
+    # Reported upstream at https://www.gaia-gis.it/fossil/librasterlite2/tktview?name=3e9183941f.
+    # Check if this can be removed with the next release.
+    inreplace "headers/rasterlite2_private.h", "#ifndef DOXYGEN_SHOULD_SKIP_THIS",
+      "#include <time.h>\n\n#ifndef DOXYGEN_SHOULD_SKIP_THIS"
+
+    # Ensure Homebrew SQLite libraries are found before the system SQLite
+    sqlite = Formula["sqlite"]
+    ENV.append "LDFLAGS", "-L#{sqlite.opt_lib} -lsqlite3"
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
```

Comment:
```suggestion
    system "./configure", *std_configure_args
```
Or does `configure` complain about `--disable-debug` and `--libdir`?

Reply:
I changed it to `*std_configure_args`.